src/Challenge.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import './style/Challenge.css';
```

Comment:
can you order ur imports as 

1. 3rd party code

2. internal code 

3. styles